content/visualize/_index.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Visualize your data"
+date = 2022-08-16T17:24:05+01:00
+weight = 5
+chapter = true
+pre = "5. "
++++
+
+
+# Visualize your Data
+
+{{<mermaid>}}
+flowchart LR
+    id1(Monitor)-->id2(Track)-->id3(Enrich)-->id4(Model)-->id5(Visualize)-->id6(Next steps)
+    style id1 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id2 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id3 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id4 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id5 fill:#f5f5f5,stroke:#6638B8,stroke-width:3px
+    style id6 fill:#f5f5f5,stroke:#333,stroke-width:1px
+{{</mermaid >}}
+
+In this chapter you can swap the data sources in the sample Tableau workbook file (twbx) to your own modeled output (csv only!) to have an overview of the current state of  website performance.
```

Comment:
double space at the end, and maybe just call out the csv thing on the next page rather than here?